etc/kayobe/kolla/config/prometheus/prometheus.rules

```diff
@@ -7,14 +7,23 @@ groups:
   rules:
 
   - alert: PrometheusTargetMissing
-    expr: up == 0
+    expr: up{job!="redfish-exporter-seed"} == 0
     for: 5m
     labels:
       severity: critical
     annotations:
       summary: "Prometheus target missing (instance {{ $labels.instance }})"
       description: "A Prometheus target has disappeared. An exporter might have crashed."
 
+  - alert: PrometheusTargetMissing
+    expr: up{job="redfish-exporter-seed"} == 0
+    for: 15m
+    labels:
+      severity: critical
+    annotations:
+      summary: "Prometheus target missing (instance {{ $labels.instance }})"
+      description: "A Prometheus target has disappeared. An exporter might have crashed."
+
```

Comment:
Duplicate of the rule above?

Reply:
Its more splitting it into two alerts, so the redfish one waits for 15m before alerting, I think?

Reply:
Ah, I hadn't noticed the expression was different. Makes sense now